Makefile

```diff
@@ -124,6 +124,11 @@ push-docker-uibackend: docker-uibackend ## Build and Push UI Backend Docker imag
 test: ## Run Unit Tests
 	@go test ./...
 
+.PHONY: end-to-end-test
+end-to-end-test: ## Run go integration test against code
```

Comment:
I feel like `make test-e2e` is much nicer to invoke

Reply:
what about `make test-end-to-end`?

Reply:
kubeclarity uses only e2e https://github.com/openclarity/kubeclarity/tree/main/e2e

Reply:
I think `e2e` makes more sense then similar to kubeclarity